api/server.js

```diff
@@ -69,7 +69,7 @@ app.get(`/ps`, (req, res) => {
 app.get(`/:routeName`, (req, res) => {
     let routeName = req.params.routeName;
     if(apiOptions[routeName] === undefined) {
-        return res.send("This route is not available.")
+        return res.send('{ "error": "This route is not available." }')
```

Comment:
the json api spec says the top level member should be named "errors". I know we've already chosen to deviate strategically from the spec; is it intended here as well?

Reply:
Anyway, merging.